draft-ietf-httpbis-message-signatures.md

```diff
@@ -244,46 +267,46 @@ The following table shows example canonicalized values for different content ide
 
 |Content Identifier|Canonicalized Value|
 |--- |--- |
-|`x-list-a:0`|()|
-|`x-list-a:1`|(a)|
-|`x-list-a:3`|(a, b, c)|
-|`x-list-a:6`|(a, b, c, d, e, f)|
-|`x-list-b:0`|()|
+|`"x-list-a";prefix=0`|()|
+|`"x-list-a";prefix=1`|(a)|
+|`"x-list-a";prefix=3`|(a, b, c)|
+|`"x-list-a";prefix=6`|(a, b, c, d, e, f)|
+|`"x-list-b";prefix=0`|()|
 {: title="Non-normative examples of list prefix canonicalization."}
 
 
-## Signature Creation Time
+## Specialty Content Fields
 
-The signature's Creation Time ({{signature-metadata}}) is identified by the `*created` identifier.
+Content not found in an HTTP header can be included in the signature base string by defining a content identifier and the canonicalization method for its content.
```

Comment:
It is not clear whether content found in an HTTP header can be further included in the signature.

Reply:
Content found in an HTTP header can be included in the signature by adding the header to the signed content, which is described in its own section.